test/Microsoft.AspNet.Mvc.ModelBinding.Test/Metadata/ModelMetadataProviderTest.cs

```diff
@@ -898,7 +898,7 @@ public AttributeInjectModelMetadataProvider(object[] attributes)
                 _attributes = attributes;
             }
 
-            protected override DefaultMetadataDetails CreateTypeDetails([NotNull]ModelMetadataIdentity key)
+            protected override DefaultMetadataDetails CreateTypeDetails(ModelMetadataIdentity key)
```

Comment:
do cleanup of unrelated files in a separate PR.  btw the reason to remove `[NotNull]` here is `ModelMetadataIdentity` is a `struct`.